### DISCUSSION TITLE
followed instructions as is on android 7.1.1 and encountered an error while applying patches.

### DISCUSSION THREAD

**Discussion by ww3459** [kitadai31/revanced-patches-android6-7#8] [General]
vanced app kitadai provided has all the permissions from the android and file system.downloaded the suggested youtube versions from the provided links in the tutorial post for android 7.1.1.process encounters an error after patching all the suggested patches and compiling resources.please see https://imgur.com/a/HdxDuFJ

**Comment by Meditacion** [comment]
Me aparece error, paso txt. no se programación y apenas entiendo, pero hice el procedimiento ayer y fallo, según entiendo problema de servidor,  hoy probé  otra vez y fallo

Initializing installer
Creating working directory
Copying original apk
Unpacking input apk
Reading dex files
Decoding AndroidManifest.xml only, because resources are not needed
Merging integrations
Deleting existing resource cache directory
Decoding resources
Applied client-spoof
Applied custom-branding-icon-afn-red
Applied custom-branding-name
Applied custom-seekbar-color
Applied custom-video-speed
Applied default-video-quality
Applied default-video-speed
Applied disable-haptic-feedback
Applied enable-external-browser
Applied enable-minimized-playback
Applied enable-new-layout
Applied enable-old-quality-layout
Applied enable-open-links-directly
Applied enable-seekbar-tapping
Applied enable-tablet-miniplayer
Applied enable-wide-searchbar
Applied force-vp9-codec
Applied header-switch
Applied hide-auto-captions
Applied hide-auto-player-popup-panels
Applied hide-autoplay-button
Applied hide-button-container
Applied hide-cast-button
Applied hide-channel-watermark
Applied hide-comment-component
Applied hide-create-button
Applied hide-crowdfunding-box
Applied hide-email-address
Applied hide-endscreen-cards
Applied hide-endscreen-overlay
Applied hide-filmstrip-overlay
Applied hide-firsttime-background-notification
Applied hide-flyout-panel
Applied hide-fullscreen-panels
Applied hide-general-ads
Applied hide-info-cards
Applied hide-live-chat-button
Applied hide-mix-playlists
Applied hide-next-prev-button
Applied hide-player-captions-button
Applied hide-player-overlay-filter
Applied hide-shorts-button
Applied hide-shorts-component
Applied hide-shorts-pivot-bar
Applied hide-snackbar
Applied hide-startup-shorts-player
Applied hide-stories
Applied hide-suggested-actions
Applied hide-time-and-seekbar
Applied hide-tooltip-content
Applied hide-video-ads
Applied layout-switch
Applied microg-support
Applied optimize-resource
Applied overlay-buttons
Applied patch-options
Applied protobuf-spoof
Applied remove-player-button-background
Applied return-youtube-dislike
Applied settings
Applied sponsorblock
Applied swipe-controls
Applied switch-create-notification
Applied theme
Applied translations
Repacking patched apk
Compiling resources
An error occurred! Aborting
Error:
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/data/app/app.revanced.manager.flutter-1/lib/arm/libaapt2.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/build/resources.zip]
	at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:365)
	at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:619)
	at app.revanced.patcher.Patcher.save(Patcher.kt:126)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda-20(MainActivity.kt:212)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$kxvfcFiIKARIH6oxYjk9gWWxjp0(MainActivity.kt)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda0.run(D8$$SyntheticClass)
	at java.lang.Thread.run(Thread.java:818)
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/data/app/app.revanced.manager.flutter-1/lib/arm/libaapt2.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/build/resources.zip]
	at brut.util.OS.exec(OS.java:99)
	at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:361)
	... 6 more

**Comment by kitadai31** [comment]
The cause is ARMv7.
Manager does not support ARMv7 (32bit) devices.
Please use another device or PC.

Or perhaps ReVanced Builder might work as well.

**Reply by ww3459** [reply]
i really wish you did  ReVanced Builder tutorial for android 7.1.1.tried to do it with termux and all that but it just opens a server link then vanced menu pops up in the server links browser page. none of the buttons are pressable  and current and advised versions just say "loading..." just cant pass through that...

**Reply by kitadai31** [reply]
Then, please try Revancify.
Yesterday, Revancify added support for our patch.
